include/GafferUI/RotateHandle.h

```diff
@@ -65,7 +65,7 @@ class GAFFERUI_API RotateHandle : public Handle
 		Imath::V3i axisMask() const;
 
 		// Measured in radians
-		Imath::Eulerf rotation( const DragDropEvent &event ) const;
+		Imath::Eulerf rotation( const DragDropEvent &event );
```

Comment:
When I introduced these Handle classes, I was trying to be frugal in terms of adding new signals, which is why I added this method and used it in conjunction with the regular `dragMoveSignal()`. But given the whole "don't forget to connect at slot 0" gotcha and this non-const "query", I think it might be time to have dedicated signals which provide the rotation as an argument. Let's have a quick chat offline as to what that might look like and whether now is the time.

Reply:
(Chatted off-line) Principal is sound, but saving for another day, when we better understand the interactions with snapping (which may well be very case-specific, eg, with options of world/local).